apps/web/src/pages/Home/Home.tsx

```diff
@@ -195,6 +195,7 @@ function Home() {
             </Button>
           </Link>
         </div>
+        <div className='absolute top-0 left-0 h-full w-full bg-black opacity-60' />
```

Comment:
why dont we make an overlay component that can be used . Custom component with color prop

Reply:
I thought of making a re-usable component but since I only used it here I decided not to make one.